iina/pl.lproj/MainMenu.strings

```diff
@@ -163,7 +163,7 @@
 "Hal-18-Tnf.title" = "Podwójny rozmiar";
 
 /* Class = "NSMenuItem"; title = "Speed Down to 0.5x"; ObjectID = "HsE-LN-Nd6"; */
-"HsE-LN-Nd6.title" = "Prędkość: 0.5x";
+"HsE-LN-Nd6.title" = "… 0.5x";
```

Comment:
It seems like you are shrinking the strings. Can you explain why you are doing this?

Reply:
It's for increase readability.

Reply:
I'm not a huge fan of doing such thing.
![screen shot 2018-06-19 at 12 46 27 am](https://user-images.githubusercontent.com/20237141/41550077-4a891e06-735a-11e8-96e1-62ac273e6b6f.png)
 But besides the readability and usability staff, the implementation there is actually quite complex. The `MenuItem.xib` localization is only part of the whole menu localization. We also ask `Localizable.strings` for some translation in the menu when appropriate. So that if you really want to change this, it will mess up the whole thing.